python/build-wheel-macos.sh

```diff
@@ -48,9 +48,6 @@ for ((i=0; i<${#PY_VERSIONS[@]}; ++i)); do
   PIP_CMD="$(dirname $PYTHON_EXE)/pip$PY_MM"
 
   pushd python
-    # Install setuptools_scm because otherwise when building the wheel for
-    # Python 3.6, we see an error.
-    $PIP_CMD install setuptools_scm
     # Fix the numpy version because this will be the oldest numpy version we can
     # support.
     $PIP_CMD install numpy==1.10.4 cython
```

Comment:
I would like to remove the `pip install cython` here, but doing so did not work out of the box.